jwst/pipeline/calwebb_spec2.py

```diff
@@ -332,7 +333,8 @@ def process_exposure_product(
             # as DO_NOT_USE or NON_SCIENCE.
             resampled = self.pixel_replace(resampled)
             resampled = self.cube_build(resampled)
-            if not self.cube_build.skip:
+            print('TYPE resampled', type(resampled))
```

Comment:
```suggestion
```

Reply:
whoops.  Is there a pre-commit tool you recommend to find these?

Reply:
Ha, no, but that does sound useful - maybe I'll look for one.  I usually just do a global search for "print(" and review all my diffs by eye.

Reply:
ok yeah I could do a better job of reviewing my diffs